pages/Drift/Drift.py

```diff
@@ -0,0 +1,105 @@
+"""
+A page of the application.
+Page content is imported from the Drift.md file.
+
+Please refer to https://docs.taipy.io/en/latest/manuals/gui/pages for more details.
+"""
+
+import taipy as tp
+from taipy.gui import Markdown
+import pandas as pd
+from taipy.gui import notify
+
+from configuration.config import scenario_cfg
+
+ref_data = pd.read_csv("data/data_ref.csv")
+
+Drift = Markdown("pages/Drift/Drift.md")
+
+tp.Core().run()
+scenario = tp.create_scenario(scenario_cfg)
+
+ref_selected = "data_ref"
+compare_selected = "data_noisy"
+
+ref_data = pd.read_csv("data/" + ref_selected + ".csv")
+scenario.reference_data.write(ref_data)
+
+compare_data = pd.read_csv("data/" + compare_selected + ".csv")
+scenario.compare_data.write(compare_data)
```

Comment:
This is application initialization. We do recommend in the documentation to put this code in the main.py, before running Gui() and in particular into the `if __name__ == "__main__": ` condition.